### PR TITLE
[RTC-78] Setup CI

### PR DESCRIPTION
**Pull request by LVala**

### CODE REVIEW

File: lib/jellyfish_web.ex [hunk 1]
```diff
@@ -50,6 +57,9 @@ defmodule JellyfishWeb do
     end
   end
 
+  @spec channel ::
+          {:use, [{:context, JellyfishWeb} | {:import, Kernel}, ...],
+           [{:__aliases__, [...], [...]}, ...]}
```
Comment (Rados13) [inline lib/jellyfish_web.ex:62]: I am not sure if these specs are useful in any way.

File: test/jellyfish_test.exs [hunk 2]
```diff
@@ -1,8 +1,4 @@
 defmodule JellyfishTest do
   use ExUnit.Case
   doctest Jellyfish
-
-  test "greets the world" do
-    assert Jellyfish.hello() == :world
-  end
 end
```
Comment (Rados13) [inline test/jellyfish_test.exs:4]: I think that currently, we could remove this file.

File: mix.exs [hunk 3]
```diff
@@ -16,16 +16,25 @@ defmodule Jellyfish.MixProject do
...
-            docs: docs()
+      # hex
+      description: "Jellyfish media server",
+      package: package(),
+
+      # docs
+      name: "Jellyfish media server",
+      source_url: @github_url,
+      homepage_url: "https://membrane.stream",
+      docs: docs(),
```
Comment (mickel8) [inline mix.exs:27]: It is very unlikely that we will use `ex_doc` for documentation, could we remove it?

File: .credo.exs [hunk 4]
```diff
@@ -31,7 +31,13 @@
           "apps/*/test/",
           "apps/*/web/"
         ],
-        excluded: [~r"/_build/", ~r"/deps/", ~r"/node_modules/"]
+        excluded: [
+          ~r"/_build/",
+          ~r"/deps/",
+          ~r"/node_modules/",
+          "lib/jellyfish_web.ex",
+          "lib/jellyfish_web/telemetry.ex"
```
Comment (mickel8) [inline .credo.exs:39]: If the reason of ignoring `jellyfish_web.ex` and `telemetry.ex` is the requirement for specs for public functions, I would favor disabling ` Credo.Check.Readability.Specs` instead

**Review by mickel8 — CHANGES_REQUESTED** [review]

**Review by Rados13 — APPROVED** [review]

File: lib/jellyfish_web/views/error_helpers.ex [hunk 5]
```diff
@@ -3,14 +3,15 @@ defmodule JellyfishWeb.ErrorHelpers do
   Conveniences for translating and building error messages.
   """
 
+  @spec translate_error({any, any}) :: any
   @doc """
   Translates an error message.
   """
   def translate_error({msg, opts}) do
```
[the remark this answers is not in the document]
Comment (LVala) [reply lib/jellyfish_web/views/error_helpers.ex:10]: I also removed `error_view.ex`. We probably won't be using templates anyway.

File: mix.exs [hunk 6]
```diff
@@ -16,16 +15,24 @@ defmodule Jellyfish.MixProject do
...
-            homepage_url: "https://membraneframework.org",
-            docs: docs()
+      # hex
+      description: "Jellyfish media server",
+      package: package(),
+
+      # docs
+      name: "Jellyfish media server",
+      source_url: @github_url,
+      homepage_url: "https://membrane.stream",
```
Comment (mickel8) [inline mix.exs:25]: Sounds like it can be remove too, can't it?

**Review by mickel8 — APPROVED** [review]

**Pull request merged**